### PR TITLE
Add Korean translation

### PR DESCRIPTION
**Pull request by billopark**
Hello! 
I translated readme along with Chinese for Korean golang users :)

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Standard Go Project Layout
 
+翻译:
```
Comment (billopark) [inline README_zh.md:3]: I just translated via google. Please check who know Chinese :-)

File: README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 Translations:
 
+* [한국어 문서](README_ko.md)
```
Comment (billopark) [inline README.md:5]: I ordered by lexicographical order of language code.

File: README_zh.md (hunk 1 above)
Comment (kcq) [reply README_zh.md:3]: hopefully somebody will be able to double check it :)

**Comment by kcq** [comment]
thanks for the PR!

**Pull request merged**